tidb-cloud/vector-search-get-started-using-python.md

```diff
@@ -54,28 +54,28 @@ pip install sqlalchemy pymysql sentence-transformers tidb-vector python-dotenv
 
 3. Ensure the configurations in the connection dialog match your operating environment.
 
-    - **Connection Type** is set to `Public`.
-    - **Branch** is set to `main`.
-    - **Connect With** is set to `SQLAlchemy`.
-    - **Operating System** matches your environment.
+   - **Connection Type** is set to `Public`.
+   - **Branch** is set to `main`.
+   - **Connect With** is set to `SQLAlchemy`.
+   - **Operating System** matches your environment.
```

Comment:
Maybe four spaces are correct.

Reply:
Automatic formatting by the very recent version of Prettier. Considering that old prettier does not format like this, I guess this is an improvement.

Reply:
I found the feature here, which is introduced 2 months ago: https://github.com/prettier/prettier/commit/6168d1e9902657d9e13c72d0795b5710cfc45100